R/class-spec.R

```diff
@@ -120,12 +140,16 @@ class_desc <- function(x) {
 
 # Vector of class names; used in method introspection
 class_dispatch <- function(x) {
+  if (identical(x, R7_object)) return(c("R7_object", "ANY"))
```

Comment:
So should the `@parent` of R7_object be `any_class`? Or maybe you can't use `any_class` as a parent in general

Reply:
I think this is actually a sign that `class_dispatch()` needs to be refactored out into two functions, because this is the base case when recursing to get the class for an R7 object. But I don't have good name for it, and this will all get rewritten in C at some point.